src/core/ApiStore.ts

```diff
@@ -160,3 +160,102 @@ export async function createUser(userToCreate: UserModel): Promise<boolean> {
 		return false;
 	}
 }
+
+export async function getCicles(): Promise<{ success: boolean; cicles: Cicle[] }> {
+	try {
+		const config = {
+			...baseConfig,
+			method: "get",
+			url: "api/cicles",
+			headers: {
+				"Content-Type": "application/json",
+			},
+		};
+
+		const response = await axios(config);
+
+		const result = {
+			success: response.status === 200,
+			cicles: response.data,
+		};
+
+		return result;
+	} catch (e) {
+		return { success: false, cicles: [] };
+	}
+}
+
+export async function getCicleQuetions(cicleId: number): Promise<{ success: boolean; cicle_questions: QuestionCategories[] }> {
+	try {
+		const config = {
+			...baseConfig,
+			method: "get",
+			url: `api/cicles/${cicleId}/questions`,
+			headers: {
+				"Content-Type": "application/json",
+			},
+		};
+
+		const response = await axios(config);
+
+		const result = {
+			success: response.status === 200,
```

Comment:
También podria ser el status 304 (Not modified), agregarlo
```suggestion
			success: [200, 304].includes(response.status),
```